kotlin/add-graphics-with-symbols/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,11 @@
+<com.esri.arcgisruntime.mapping.view.MapView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/mapView"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
```

Comment:
Since there is only one view in the activity, there is no need for an outer constraint layout, which I think @TADraeseke advised me of previously. In addition, I think we can remove these margins, imho it looks better without them. This is the only time I have seen margins in a sample so it did seem odd to have them. I am removing them. Please advise if there is a reason to bring them back in.